javascript/frameworks/cap/lib/advanced_security/javascript/frameworks/cap/RemoteFlowSources.qll

```diff
@@ -64,3 +68,46 @@ class ServiceinCDSHandlerParameter extends ParameterNode, RemoteFlowSource {
     result = "Parameter of an event handler belonging to an exposed service defined in a cds file"
   }
 }
+
+/**
+ * A parameter of a handler registered for a service on an event. e.g.
+ * ```javascript
+ * cds.serve('./test-service').with((srv) => {
+ *  srv.before('READ', '*', (req) => req.reply([]))
+ * })
+ * ```
+ * The parameter named `req` is captured in the above example.
+ *
+ * DOES NOT REQUIRE that a `UserDefinedApplicationService` is explicitly defined
+ * DOES NOT REQUIRE that the name is provided explicitly
```

Comment:
Although these contract specifications are very helpful, they are a bit out of style with the rest of the comments. Can you smooth these out into a full sentence?